course/lesson-5-your-first-function/lesson.tres

```diff
@@ -147,7 +147,7 @@ title = ""
 type = 0
 text = "The instructions inside the function [b]must[/b] all start with a leading tab character. You can insert that tab character by pressing the [b]Tab[/b] key.
 
-We call those leading tabs [i]indents[/i]. They're important: the computer uses them to know which instructions are part of the same code block."
+We call those leading tabs [i]indents[/i]. They're important: the computer uses them to know which instructions are part of the same group."
```

Comment:
You change `code block` only once in the course but we use it in different places. I think this should be consistent, and explained if it's unclear.

A group of instructions might refer to an entire function with multiple indent levels or just a bunch of lines that are grouped together visually but that are not part of a block. Block has a specific meaning (code at the same indent level or between brackets) and it's terminology people can use online. Though I checked and right now, the expression's used in the app but not explained.